cmd/ak/cmd/manifest/exec.go

```diff
@@ -0,0 +1,39 @@
+package manifest
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"go.autokitteh.dev/autokitteh/cmd/ak/common"
+	"go.autokitteh.dev/autokitteh/sdk/sdkmanifest"
+)
+
+var execCmd = common.StandardCommand(&cobra.Command{
```

Comment:
I don't understand, why have both `apply` and `exec`? Also, "execute" is an inaccurate name from this CLI command - we're not executing code in the regular sense, we're applying configurations.

Reply:
apply is for:
- simple single command to do everything
- backward compatible to what we have now

exec is executing actions, just executes all the actions one after another.

Reply:
Worth thinking about it. Now in ENG-450.